build.sbt

```
@@ -78,6 +78,10 @@ ThisBuild / githubWorkflowBuild := Seq(
     cond = Some(s"matrix.ci == 'ciJS' && matrix.os == '$PrimaryOS'")))
 
 ThisBuild / githubWorkflowBuildMatrixAdditions += "ci" -> List("ciJVM", "ciJS", "ciFirefox")
+ThisBuild / githubWorkflowBuildMatrixInclusions += MatrixInclude(
+  Map("scala" -> Scala213),
+  Map("os" -> Windows)
+)
```

The trick here is you have to include *all* keys. So you probably want the following:

```suggestion
ThisBuild / githubWorkflowBuildMatrixInclusions += MatrixInclude(
  Map("scala" -> Scala213, "java" -> PrimaryJava, "ci" -> "ciJVM"),
  Map("os" -> Windows)
)
```